src/views/home/components/token-selector-header/token-selector-header.view.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+
+import useTokenSelectorHeaderStyles from "src/views/home/components/token-selector-header/token-selector-header.styles";
+import { ReactComponent as ArrowLeftIcon } from "src/assets/icons/arrow-left.svg";
+import { ReactComponent as XMarkIcon } from "src/assets/icons/xmark.svg";
+import Typography from "src/views/shared/typography/typography.view";
+
+interface TokenSelectorHeaderProps {
+  title: string;
+  onClose?: () => void;
```

Comment:
Why is `onClose` optional? I don't think it has to be optional, but if it's, then the button should render conditionally, otherwise, you see it, click it and it does nothing.

Reply:
Right, forgot to add the conditional render, fixed!